### DISCUSSION TITLE
Cmdlet to identify duplicate model ids

### DISCUSSION THREAD

**Discussion by FH-Inway** [d365collaborative/d365fo.tools#752] [Ideas]
Artem Tymofyeyev describes in their blog a PowerShell script to identify duplicate model ids. This happens when Microsoft adds new modules, which can conflict with customizations or ISV solutions that already use that id. Conflict could also happen between ISV solutions from 2 different ISVs, but I have not heard of such a case yet.

https://artemtymofyeyev.blogspot.com/2020/10/d365fo-duplicate-model-ids.html

Latest occurence of this in https://www.yammer.com/dynamicsaxfeedbackprograms/threads/2415733453299712

Idea is to add a cmdlet to the tools that identifies the duplicate model ids. Should ask Artem first before we reuse their code.

**Comment by Splaxi** [comment]
Great idea. Lets see if he would be willing to have his code included.
